clientlibrary/worker/worker.go

```diff
@@ -292,7 +292,7 @@ func (w *Worker) eventLoop() {
 				if err != nil {
 					// checkpoint may not existed yet is not an error condition.
 					if err != chk.ErrSequenceIDNotFound {
-						log.Errorf(" Error: %+v", err)
+						log.Warnf("Couldn't fetch checkpoint: %s", err)
```

Comment:
How about keeping the %+v.

Reply:
yes, it's ok for me, I will revert the change.